pkg/apis/serving/v1beta1/inference_service_status.go

```diff
@@ -65,6 +68,9 @@ type ComponentStatusSpec struct {
 	// It generally has the form http[s]://{route-name}.{route-namespace}.{cluster-level-suffix}
 	// +optional
 	URL *apis.URL `json:"url,omitempty"`
+	// gRPC endpoint of the component if different from URL.
+	// +optional
+	GrpcURL *apis.URL `json:"grpcUrl,omitempty"`
```

Comment:
So the thinking is we only fill this in if different from `URL`?

This seems good but I guess what's missing is a way to tell which of REST/gRPC are exposed in all cases. I.e. when `GrpcURL` isn't present, does `URL` indicate REST or gRPC or both?

Reply:
Hmm, yea. I Haven't quite settled on how to handle this. Leaning towards my original approach of having both `HttpURL` and `GrpcURL`. And if they are the same, then it'll just be populated twice in both fields.